hardware/arduino/avr/libraries/EEPROM/README.md

```diff
@@ -128,12 +128,12 @@ ptr++; //Move to next EEPROM cell.
 
 #### **`EEPROM.begin()`**
 
-This function returns an `EEPtr` pointing to the first cell in the EEPROM.  
```

Comment:
This trailing whitespace was intentional but I don't think the readability is significantly harmed by its removal.